README.md

```diff
@@ -7,7 +7,7 @@
 
 ## PacketFrame
 
-The PacketFrame CDN is an open source (AGPL-3.0) anycast CDN platform, currently in beta. If you're interested in an account, send an email to the address listed on the front page of https://packetframe.com or reach out in #packetframe on freenode.
+The PacketFrame CDN is an open source (AGPL-3.0) anycast CDN platform, currently in beta. If you're interested in an account, send an email to the address listed on the front page of https://packetframe.com or reach out in #packetframe on the [Libera.Chat](https://freenode.net/kb/answer/chat) IRC network.
```

Comment:
Let's update the link to libera.chat here

Reply:
Done, I think 👯